lib/CIGateway.py

```diff
@@ -27,11 +27,14 @@ def start(cls, jenkins_url, jenkins_user, jenkins_pass, user='jenkins'):
                    'charm_dir': hookenv.charm_dir()
                })
 
+        subprocess.check_call(['systemctl', 'enable', 'cwr-server'])
         subprocess.check_call(['systemctl', 'daemon-reload'])
         host.service_start('cwr-server')
```

Comment:
Please use charmhelpers.core.host.service_x methods so we don't hate ourselves if we ever need to support something besides systemd:

http://bazaar.launchpad.net/~charm-helpers/charm-helpers/devel/view/head:/charmhelpers/core/host.py#L59

```python
charmhelpers.core.host.service_enable('cwr-server')
if charmhelpers.core.host.init_is_systemd():
    subprocess.check_call(['systemctl', 'daemon-reload'])
charmhelpers.core.host.service_start('cwr-server')
```